ui/src/features/stage/stage-details.tsx

```diff
@@ -92,7 +51,21 @@ export const StageDetails = () => {
               subscriptions={data.stage.spec?.subscriptions}
               projectName={projectName}
             />
-            <AvailableFreight stage={data.stage} onSuccess={refetch} />
+            <Tabs
+              defaultActiveKey='1'
+              items={[
+                {
+                  key: '1',
+                  label: 'Available Freight',
+                  children: <AvailableFreight stage={data.stage} onSuccess={refetch} />
+                },
+                {
+                  key: '2',
+                  label: 'Manifest Preview',
```

Comment:
No need for the word "preview". I suggest:

```suggestion
                  label: 'Manifest',
```

Reply:
Or Live Manifest (which is what we use in Argo CD)